content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Retroactive default StorageClass"
+date: 2022-11-11
```

Comment:
It should be 1"Kubernetes 1.26" and the date should be somewhere after 1.26  GA.  We want it as post-1.26 feature blog

Reply:
I see, the "Release blog published" date is 6th December 2022 so that date might make the most sense here.